.github/workflows/native-aot.yml

```diff
@@ -83,7 +83,7 @@ jobs:
           size="$(ls -l test/Npgsql.NativeAotTests/bin/Release/net8.0/linux-x64/native/Npgsql.NativeAotTests | cut -d ' ' -f 5)"
           echo "Binary size is $size bytes ($((size / (1024 * 1024))) mb)"
 
-          if (( size > 15728640 )); then
-            echo "Binary size exceeds 15mb threshold"
+          if (( size > 7340032 )); then
```

Comment:
:tada:

Reply:
Yeah, we currently at around 5.8mb, so 7mb should be quite enough.

Reply:
To zero... and beyond?